engine/common/follower/engine.go

```diff
@@ -344,19 +347,17 @@ func (e *Engine) processBlockProposal(ctx context.Context, proposal *messages.Bl
 	// it only checks the block header, since checking block body is expensive.
 	// The full block check is done by the consensus participants.
 	err := e.state.Extend(ctx, block)
-	// if the error is a known invalid extension of the protocol state, then
-	// the input is invalid
-	if state.IsInvalidExtensionError(err) {
-		return engine.NewInvalidInputErrorf("invalid extension of protocol state: %w", err)
-	}
-
-	// if the error is a known outdated extension of the protocol state, then
-	// the input is outdated
-	if state.IsOutdatedExtensionError(err) {
-		return engine.NewOutdatedInputErrorf("outdated extension of protocol state: %w", err)
-	}
-
 	if err != nil {
+		// The proposal must represent a valid extension of the protocol state, otherwise it is invalid.
```

Comment:
It might be clearer to use the same log levels and messages here as the other engines where this case is handled:
https://github.com/onflow/flow-go/blob/5d1c56881355211d1502c12de749fb832ae99ac8/engine/consensus/compliance/core.go#L253-L262

Reply:
👍 implemented in https://github.com/onflow/flow-go/pull/2372/commits/412fd2a54fa706a5773c9e4bf3ea58d646ae546f

Only comment: it is not _exactly_ the same log message, as the same symptom has different explanations from the view point of the node dealing with the orphaned block